src/Microsoft.Health.Fhir.Core.UnitTests/Models/PartialDateTimeTests.cs

```diff
@@ -411,62 +348,18 @@ public void GivenAValidPartialDateTime_WhenToStringIsCalled_ThenCorrectStringSho
         [InlineData("de-DE", "2018-11-29T18:30:27.911+01:00", "2018-11-29T18:30:27,9110000+01:00")]
         [InlineData("en-GB", "2018-11-29T18:30:27.911+01:00", "2018-11-29T18:30:27.9110000+01:00")]
         [InlineData("en-US", "2018-11-29T18:30:27.911+01:00", "2018-11-29T18:30:27.9110000+01:00")]
-        public void GivenACulture_WhenToStringisCalled_ThenCorrectStringShouldBeReturned(string culture, string input, string expected)
+        public void GivenACulture_WhenToStringIsCalled_ThenCorrectStringShouldBeReturned(string culture, string inputString, string expectedString)
         {
-            System.Threading.Thread.CurrentThread.CurrentCulture = new System.Globalization.CultureInfo(culture);
-            PartialDateTime dateTime = PartialDateTime.Parse(input);
+            System.Threading.Thread.CurrentThread.CurrentCulture = new CultureInfo(culture);
```

Comment:
You could use the `Mock.Property` helper here to ensure the thread is set back to the original value:
```
using (Mock.Property(() => System.Threading.Thread.CurrentThread.CurrentCulture, new CultureInfo(culture)))
{
 //...
}
```

Reply:
Ah, already covered, nvm :)